server/public/partials/statblock/statBlock.html

```diff
@@ -153,8 +153,15 @@ <h4>Languages</h4>
         </p>
       </div>
       <div class="property-line">
-        <h4>Challenge</h4>
-        <p>{{creature.stats.challengeRatingStr}} ({{creature.stats.experiencePoints}} XP)</p>
+        <h4 style="opacity: 0;">Challenge</h4>
```

Comment:
I'd be wary of using opacity like this. Seems like it might cause issues on different screen/device sizes, or in the future as we continue to modify the UI. What's the goal with the invisible "Challenge"? To create space? Can you use margin or padding to accomplish the same thing?

Reply:
Yeah, you are totally right. I'm not used to working on UI and so was (somewhat comically) unware that you can create a margin in a vertical plane as well.... I'll change it